synapse/cores/common.py

```diff
@@ -1901,6 +1901,33 @@ def formTufoByTufo(self, tufo):
 
         return self.formTufoByProp(form, valu, **props)
 
+    def formTufosByItems(self, items):
+        '''
+        Forms tufos by prop, given a tuple of (form, valu, props) tuples.
+
+        Args:
+
+            items (tuple): A tuple of tuples of (form, valu, props)
+
+        Returns:
+            tuple: Tuple containing tufos, either with the node or error data
+
+        Example:
+
+            items = ( ('foo:thing', 'hehe', {'a': 1}), ('bar:thing', 'haha', {'b': 2}), )
+            results = core.formTufoByItems(items)
+        '''
+        retval = []
+
+        with self.getCoreXact() as xact:
+            for form, valu, props in items:
+                try:
+                    retval.append(self.formTufoByProp(form, valu, **props))
+                except Exception as e:
+                    retval.append(('error', s_common.excinfo(e)),)
```

Comment:
this could probably use the "ephemeral tufo" convention ( None as iden ) so it's consistent with other interfaces.  Additionally, we should probably insert the form/valu/props here as well so we know which one caused the error. See synapse/lib/tufo.py  def ephem() for a helper